core/script_language.cpp

```diff
@@ -563,7 +563,8 @@ Variant PlaceHolderScriptInstance::property_get_fallback(const StringName &p_nam
 PlaceHolderScriptInstance::PlaceHolderScriptInstance(ScriptLanguage *p_language, Ref<Script> p_script, Object *p_owner) :
 		owner(p_owner),
 		language(p_language),
-		script(p_script) {
+		script(p_script),
+		build_failed(false) {
```

Comment:
Now that I think of it, It could be a good idea to change `build_failed` to an abstract method in `Script`. It might be cleaner this way, avoiding the loops for updating its value in placeholders.

Reply:
Makes sense to me. CC @vnen